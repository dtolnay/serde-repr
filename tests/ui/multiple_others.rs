use serde_repr::DeserializeRepr;

#[derive(DeserializeRepr)]
#[repr(u8)]
enum MultipleOthers {
    #[serde(other)]
    A,
    #[serde(other)]
    B,
}

fn main() {}
