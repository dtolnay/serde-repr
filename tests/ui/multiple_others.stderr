error: only one variant can be #[serde(other)]
 --> $DIR/multiple_others.rs:3:10
  |
3 | #[derive(DeserializeRepr)]
  |          ^^^^^^^^^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
